src/codemagic/tools/app_store_connect/action_groups/builds_action_group.py

```diff
@@ -71,6 +75,45 @@ def get_build(self, build_id: ResourceId, should_print: bool = True) -> Build:
 
         return self._get_resource(build_id, self.api_client.builds, should_print)
 
+    @cli.action(
+        "list",
+        AppArgument.APPLICATION_ID_RESOURCE_ID_OPTIONAL,
+        *ArgumentGroups.LIST_BUILDS_FILTERING_ARGUMENTS,
+        action_group=AppStoreConnectActionGroup.BUILDS,
+        deprecation_info=cli.ActionDeprecationInfo("list-builds", "0.49.0"),
+    )
+    def list_builds(
```

Comment:
Used to be defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L267-L297